pio-tools/gen-berry-structures.py

```diff
@@ -0,0 +1,12 @@
+Import("env")
+import os
+import subprocess
+from os.path import join
+
+# generate all precompiled Berry structures from multiple modules
+CURRENT_DIR = os.getcwd()
+BERRY_GEN_DIR = join(CURRENT_DIR, "lib", "libesp32","berry")
```

Comment:
BERRY_GEN_DIR = join(env.subst("$PROJECT_DIR"), "lib", "libesp32","berry")

Reply:
Yes, we can do it like that.
I am not 100% sure if it really is important to go back to the current directory at the end, but it should not hurt.